packages/collections/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@monument/collections",
+  "version": "0.15.0",
+  "description": "TypeScript collections",
+  "keywords": [
+    "typescript",
+    "collections",
+    "map",
+    "list",
+    "array",
+    "queue",
+    "stack",
+    "set",
+    "tree",
+    "standard",
+    "library",
+    "core"
+  ],
+  "license": "MIT",
+  "homepage": "https://monumentjs.github.io/package/core",
+  "repository": "https://github.com/monumentjs/core.git",
+  "contributors": [
+    {
+      "name": "Alex Chugaev",
+      "email": "achugaev93@gmail.com"
+    }
+  ],
+  "dependencies": {
+    "@monument/core": "^0.15.0",
+    "reflect-metadata": "^0.1.13"
```

Comment:
Not sure "reflect-metadata" is used by collections. Please check. If it unused, remove this dependency.

Reply:
Yeah - it's not used. Removed it :ok_hand: